modules/core/src/v2/coins/eos.ts

```diff
@@ -9,25 +9,31 @@ import {
   KeyPair,
   ParseTransactionOptions,
   ParsedTransaction,
-  VerifyTransactionOptions,
   VerifyAddressOptions as BaseVerifyAddressOptions,
   HalfSignedAccountTransaction as BaseHalfSignedTransaction,
   SignTransactionOptions as BaseSignTransactionOptions,
+  VerificationOptions,
+  VerifyTransactionOptions as BaseVerifyTransactionOptions,
 } from '../baseCoin';
 import { NodeCallback } from '../types';
 import { BigNumber } from 'bignumber.js';
 import { randomBytes } from 'crypto';
+// TODO BG-36375: remove dependency on eosjs v16
```

Comment:
Please see comment above. I think this shall be resolved before releasing into production.